### PR TITLE
Shared FCE Count fix

### PR DESCRIPTION
**Pull request by christinem**
This fixes issue #592.

### CODE REVIEW

File: public/js/graph/setup.js [hunk 1]
```diff
@@ -2,13 +2,17 @@
 // Globals
 var timetable = 'static/res/timetable.html';
 var clickedCourses = [];
-var FCEs = 0;
+
+// FCEs for current graph
+var FCEs = 0;
```
Comment (david-yz-liu) [inline public/js/graph/setup.js]: Maybe this doesn't need to be global.

Comment (christinem) [inline public/js/graph/setup.js]: We could leave `totalFCEs` as the only global variable and just increase the credit count with that variable. Do you think we have any need of keeping track of credit counts for individual graphs? Right now I store those as cookies, but if not I could make it simple and remove that code, and keep `totalFCEs` as the only global. It's really the only reason `FCEs` is still global.

Comment (david-yz-liu) [inline public/js/graph/setup.js]: I see. If `FCEs` tracks the current FCEs selected on the current department, then why don't you:

1) rename the variables
2) display its value alongside the total FCE count (perhaps just in parentheses after the global total, for now)

**Comment by christinem** [comment]
@david-yz-liu I've made it so that the current graph FCE total is in brackets beside the global FCE count (although I'm not sure how clear it is to the user's eye). Pressing "Reset Graph" only resets the FCEs of the current graph, and adjusts the global value accordingly to whatever credits are left selected in the other graphs. Is that functionality okay?

**Comment by david-yz-liu** [comment]
Thanks, @christinem! The behaviour of the "Reset Graph" is fine. I disabled the total from actually appearing (even though it is computed) because some courses are shared between graphs, and they would be counted twice, which isn't great. A far more complicated solution is required for that.

**Pull request merged**